docs/support/contact_us.md

```diff
@@ -0,0 +1,9 @@
+---
+title: Contact Us
+category: 6255e67693d5200013b1fa42
```

Comment:
Is this new page meant to _replace_ the current [contact us page](https://docs.phylum.io/docs/support)?

Reply:
Yes, it will. Was making a change to it, so decided to bring it under GH control.